### PR TITLE
Pass tests

### PR DESCRIPTION
**Pull request by lclarksl**

### CODE REVIEW

File: tests/unit/services/speaker.service.spec.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
-describe('BristechSRM', function() {
+describe('Speaker service', function() {
     var service, $httpBackend;
 
-    var backend = {url : 'http://localhost', port: 9000};
+    var backend = {url : 'http://api.bris.tech', port: 80};
```
Comment (jebbinSCL) [inline tests/unit/services/speaker.service.spec.js]: Did we not say that we would get this from the config?

**Comment by thullSL** [comment]
Once json's point has been answered :shipit:

**Comment by lclarksl** [comment]
Changed it so it gets the url from config

**Comment by jebbinSCL** [comment]
:shipit:

**Pull request merged**